docs/lessons/lesson1-10/Boltdir/.rerun.json

```diff
@@ -0,0 +1 @@
+[{"target":"node1","status":"success"},{"target":"node2","status":"failure"},{"target":"node3","status":"failure"},{"target":"winrm://localhost:55985","status":"failure"}]
```

Comment:
I don't think we want to include rerun files, since they don't seem to be part of the lab